src/lib/api/feed-manip.ts

```diff
@@ -103,7 +111,7 @@ export class FeedViewPostsSlice {
     })
     if (isGrandparentBlocked) {
       this.isOrphan = true
-      // Keep going, it might still have a root.
+      return
```

Comment:
i'm not sure this is correct — this would make it have no root which would e.g. break thread deduping. is this necessary?